src/Entity/Committee.php

```diff
@@ -167,6 +167,13 @@ class Committee extends BaseGroup implements SynchronizedEntity, ReferentTaggabl
      */
     private $mailchimpId;
 
+    /**
+     * @var CommitteeElection|null
+     *
+     * @ORM\OneToOne(targetEntity="CommitteeElection", mappedBy="committee")
```

Comment:
after you explain [here](https://github.com/EnMarche/en-marche.fr/pull/4276/files#diff-203adc045a327c97d8c3f53d4fa041e5R23) to the database the cascade between committees & elections, now explain it to the ORM by adding `cascade={"all"}, orphanRemoval=true` here